### DISCUSSION TITLE
Resources in composer package

### DISCUSSION THREAD

**Discussion by kvostyc** [filamentphp/filament#8001] [Help] [answered]
Hey, im trying to make some package, and my question is.. How to invoke resources, or other filament components from package to base Laravel filament installation ?

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/3.x/panels/plugins#configuring-the-panel-with-a-plugin-class

**Reply by PieterxJan** [reply]
Hi @danharrin,

I'm currently updating from Filament v2 to v3 and I came up against this "issue". 
Is it correct that it isn't possible anymore in v3 to invoke a resource to show up, without having to add the Plugin to the panel? 

In v2 I just created a function like this:
```php
protected function getResources(): array
{
    return [
        UserResource::class,
    ];
}
```

Ideally, I'd like the panels to automatically discover the defined resources, **over several packages**, so it shows up on the admin panel. Is there any way to do that? 

Kind regards,
PJ
